quodlibet/browsers/covergrid/models.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2022 Thomas Leberbauer
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+from gi.repository import GObject, Gio
+
+from quodlibet import _, ngettext, app
+from quodlibet.qltk.models import ObjectModelSort, ObjectStore, ObjectModelFilter
+from quodlibet.util.library import background_filter
+
+
+class AlbumListItem(GObject.Object):
+
+    def __init__(self, album=None):
+        super().__init__()
+        self._album = album
+        self._cover = None
+        self._label = None
+
+        self.connect('notify::album', self._album_changed)
+
+    def load_cover(self, size, cancelable=None):
+        def callback(cover):
+            self._cover = cover
+            self.notify('cover')
+
+        manager = app.cover_manager
+        # Skips this during testing
+        if manager:
+            manager.get_pixbuf_many_async(
+                self._album.songs, size, size, cancelable, callback)
+
+    def format_label(self, pattern):
+        self._label = pattern % self._album
+        self.notify('label')
+
+    def _album_changed(self, model, prop):
+        self._label = None
+
+    @GObject.Property
+    def album(self):
+        return self._album
+
+    @GObject.Property
+    def cover(self):
+        return self._cover
+
+    @GObject.Property
+    def label(self):
+        return self._label
+
+
+class AlbumListCountItem(AlbumListItem):
+
+    def load_cover(self, *args, **kwargs):
+        self.notify('cover')
+
+    def format_label(self, pattern=None):
+        n = self.__n_albums
+        self._label = ('<b>%s</b>\n' +
+            ngettext('%d album', '%d albums', n)) % (_('All Albums'), n)
```

Comment:
We should escape that really, in case translations end up with markup

Reply:
Are you referring to this? https://github.com/quodlibet/quodlibet/blob/fec3e52ee97014b7f692314b581ae44fa7817485/quodlibet/util/__init__.py#L203-L205

Reply:
That's the one!